frontend/src/components/channels/ChannelDropdownMenu.tsx

```diff
@@ -76,6 +78,23 @@ export function ChannelDropdownMenu({
               </DropdownMenuItem>
             </AlertDialogTrigger>
           )}
+          {channel.localSpendableBalance > 0 && (
+            <DropdownMenuItem
+              className="flex flex-row items-center gap-2 cursor-pointer"
+              onClick={() => {
+                const amount = channel.localSpendableBalance / 1000;
```

Comment:
We currently cannot choose what channel to pay from, so I think this option needs to go in the advanced dropdown for the node page itself, rather than per channel.

We can probably not spend the whole amount. Would it make sense to allow the user to input an amount, and default to ~90% of the value of the largest channel? that way if it fails, users can still try again with a different amount.

Reply:
> We currently cannot choose what channel to pay from, so I think this option needs to go in the advanced dropdown for the node page itself, rather than per channel.

> We can probably not spend the whole amount. Would it make sense to allow the user to input an amount, and default to ~90% of the value of the largest channel? that way if it fails, users can still try again with a different amount.

Oh, okay. Thanks for this feedback, much appreciated :muscle: